nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -538,23 +547,22 @@ contract StakingEscrow is Issuer {
     * @param _periods Amount of periods during which tokens will be locked
     **/
     function lock(address _staker, uint256 _value, uint16 _periods) internal {
-        require(_value <= token.balanceOf(address(this)) &&
-            _value >= minAllowableLockedTokens &&
+        require(_value >= minAllowableLockedTokens &&
             _periods >= minLockedPeriods);
 
         uint16 currentPeriod = getCurrentPeriod();
-        uint16 nextPeriod = currentPeriod.add16(1);
+        uint16 nextPeriod = currentPeriod + 1;
         StakerInfo storage info = stakerInfo[_staker];
         uint256 lockedTokens = getLockedTokens(info, currentPeriod, nextPeriod);
-        require(_value.add(lockedTokens) <= info.value &&
-            _value.add(lockedTokens) <= maxAllowableLockedTokens);
+        uint256 requestedLockedTokens = _value.add(lockedTokens);
```

Comment:
Change `add()` by `+`?

Reply:
here `_value` comes from outside, if somebody set too high `_value` then will be overflow